aspnetcore/fundamentals/servers/httpsys.md

```diff
@@ -1,179 +1,168 @@
-
-* Preregister URL prefixes to bind to HTTP.sys, and set up SSL certificates
-
-   If you don't preregister URL prefixes in Windows, you have to run your application with administrator privileges. The only exception is if you bind to localhost using HTTP (not HTTPS) with a port number greater than 1024; in that case, administrator privileges aren't required.
-
-   For details, see [How to preregister prefixes and configure SSL](#preregister-url-prefixes-and-configure-ssl) later in this article.
-
-* Open firewall ports to allow traffic to reach HTTP.sys.
+HTTP.sys is useful for deployments where:
 
-   You can use *netsh.exe* or [PowerShell cmdlets](https://technet.microsoft.com/library/jj554906).
+* There's a need to expose the server directly to the Internet without using IIS.
 
-There are also [Http.Sys registry settings](https://support.microsoft.com/kb/820129).
+  ![HTTP.sys communicates directly with the Internet](httpsys/_static/httpsys-to-internet.png)
 
-### Configure your ASP.NET Core application to use HTTP.sys
+* An internal deployment requires a feature not available in Kestrel, such as [Windows Authentication](xref:security/authentication/windowsauth).
 
-* No package install is needed if you use the [Microsoft.AspNetCore.All](xref:fundamentals/metapackage) metapackage. The [Microsoft.AspNetCore.Server.HttpSys](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.HttpSys/) package is included in the metapackage.
+  ![HTTP.sys communicates directly with the internal network](httpsys/_static/httpsys-to-internal.png)
 
-* Call the `UseHttpSys` extension method on `WebHostBuilder` in your `Main` method, specifying any [HTTP.sys options](https://github.com/aspnet/HttpSysServer/blob/rel/2.0.0/src/Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs) that you need, as shown in the following example:
+HTTP.sys is mature technology that protects against many types of attacks and provides the robustness, security, and scalability of a full-featured web server. IIS itself runs as an HTTP listener on top of HTTP.sys. 
 
-  [!code-csharp[](httpsys/sample/Program.cs?name=snippet_Main&highlight=11-19)]
-
-### Configure HTTP.sys options
-
-Here are some of the HTTP.sys settings and limits that you can configure.
+## How to use HTTP.sys
 
-**Maximum client connections**
+### Configure the ASP.NET Core app to use HTTP.sys
 
-The maximum number of concurrent open TCP connections can be set for the entire application with the following code in *Program.cs*:
+1. A package reference in the project file isn't required when using the [Microsoft.AspNetCore.All metapackage](xref:fundamentals/metapackage) ([nuget.org](https://www.nuget.org/packages/Microsoft.AspNetCore.All/)) (ASP.NET Core 2.0 or later). When not using the `Microsoft.AspNetCore.All` metapackage, add a package reference to [Microsoft.AspNetCore.Server.HttpSys](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.HttpSys/).
```

Comment:
It's best to just say "metapackage", since there's a new "Microsoft.AspNetCore.App" metapackage in 2.1.

Reply:
Since there hasn't been agreement on the language yet for [Replace ALL meta-package wit App Microsoft.AspNetCore.App - MD files only (\#5314)](https://github.com/aspnet/Docs/issues/5314), perhaps we should wait. This topic will get swept up in the search for ".All" on the repo, so we know it will get addressed when #5314 is tackled.